### ISSUE TITLE
Instance: Focus is blurred when the "People" listbox is closed via the `Escape` key | WCAG 2.2 Violation - There are instances of incorrect/missing focus movement (after user-initiated actions) present within the interface.

### ISSUE THREAD

**Issue by ivrouwe-ubc** [ubc/wp-comments-chat-ui#52]
#### This violates the following WCAG (Web Content Accessibility Guidelines) 2.2 Success Criteria:

- [2.4.3 Focus Order](https://www.w3.org/TR/WCAG22/#focus-order)

#### Further information:

1. 

### Screenshot

![Screenshot of the 'People' listbox.](https://github.com/user-attachments/assets/8996b350-fd00-429e-b894-420a24843b21)

### Code Reference

```
<!-- The "People" listbox -->
<dialog class="mention-dialog" aria-label="Select a person to mention">
	[…]
	<ul class="mention-list" role="listbox" aria-label="Mentionable users">[…]</ul>
	[…]
</dialog>
```
